server/server/Controllers/AccountController.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using server.Models;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.EntityFrameworkCore;
+using server.DBRepositories;
+
+namespace server.Controllers
+{
+    public class AccountController : Controller
+    {
+        private UnitOfWork db;
+        public AccountController(ApplicationContext context)
+        {
+              db = new UnitOfWork(context);
+        }
+
+        [Route("/role")]
+        [HttpGet]
+        public string GetRole()
+        {
+            string userEmail = HttpContext.User.Identity.Name;
```

Comment:
Better to save Users and admins in the same table to reduce quantity of queries to db